csi/controllerserver.py

```diff
@@ -70,6 +70,26 @@ def execute_gluster_quota_command(privkey, user, host, gvolname, path, size):
     return None
 
 
+def is_raw_block_pvc(request):
+    """If `volumeMode: Block` is specified in PVC"""
+    block = False
+    for vol_capability in request.volume_capabilities:
+        logging.info("HERE: ")
+        logging.info(vol_capability.access_type)
+        logging.info(csi_pb2.VolumeCapability.BlockVolume)
+        if vol_capability.access_type == csi_pb2.VolumeCapability.BlockVolume:
```

Comment:
```suggestion
        if vol_capability.get('access_type', '') == csi_pb2.VolumeCapability.BlockVolume:
```

would that resolve the problem? Looks like for block it is getting 'access_type as the option, but for everything else, its not having this parameter in argument, hence I suspect the failure is?

Reply:
It is kind of different. "access_type" is `oneof` type in volume context. It can have either block or mount type.

```
// Specifies what API the volume will be accessed using. One of the
// following fields MUST be specified.
  oneof access_type {
    BlockVolume block = 1;
    MountVolume mount = 2;
  }
```

This doc suggests to use `WhichOneof` to find the name.

https://developers.google.com/protocol-buffers/docs/reference/python-generated#oneof